### PR TITLE
bug: fix supporting the Patch method and logging the body

### PR DESCRIPTION
**Pull request by alastairtree**
If you need to mock an API designed to the JsonAPI specification you need to work with PATCH requests with a http body. This fix and tests should allow support of PATCH with body and ensure the correct information gets logged.

- Patch requests not configured to support Body
- Add a unit test covering a patch request and logging of results

### CODE REVIEW

File: src/WireMock.Net/Owin/OwinRequestMapper.cs [hunk 1]
```diff
@@ -91,8 +91,9 @@ private bool ParseBody(string method)
                 TRACE - Body not supported.
                 OPTIONS - Body supported but no semantics on usage (maybe in the future).
                 CONNECT - No defined body semantics
+                PATHC - Body suported
```
Comment (StefH) [inline src/WireMock.Net/Owin/OwinRequestMapper.cs]: PATCH

Comment (alastairtree) [reply src/WireMock.Net/Owin/OwinRequestMapper.cs]: Oops. Fixed

**Comment by alastairtree** [comment]
Not sure but don't think this has been release? Could it be please?

**Comment by StefH** [comment]
Done

**Pull request merged**